### PR TITLE
remove piix4-uhic usb controller from window

### PR DESCRIPTION
**Pull request by michiboo**
id: 1714159

Signed-off-by: Micky Yun Chan <michan@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/usb.cfg [hunk 1]
```diff
@@ -21,6 +21,8 @@
             max_ports_usbtest = 2
             drive_format_stg = "usb1"
             no ppc64 ppc64le
+            Host_RHEL:
```
Comment (vivianQizhu) [inline qemu/tests/cfg/usb.cfg]: This line is not necessary.

File: qemu/tests/cfg/usb.cfg [hunk 2]
```diff
@@ -21,6 +21,8 @@
...
+                no Win2000, WinXP, Win2003, WinVista, Win7, Win2008
```
Comment (vivianQizhu) [inline qemu/tests/cfg/usb.cfg]: `no Windows` is enough.

**Comment by vivianQizhu** [comment]
@XueqiangWei Request for your review, thanks.

**Comment by XueqiangWei** [comment]
dryrun \"usb.usb_hotplug.usb_multi_times.usb_normal_test.usb_ccid.with_usb_hub.one_usb_dev.piix4-uhci\" for linux and windows guest, get linux one. So it works well. 

python ConfigTest.py --testcase=usb.usb_hotplug.usb_multi_times.usb_normal_test.usb_ccid.with_usb_hub.one_usb_dev.piix4-uhci --guestname=RHEL.7.8,Win2019 --platform=x86_64 --machine=q35 --firmware=ovmf --clone=no --dryrun
 (1/1) Host_RHEL.m7.u8.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.usb.usb_hotplug.usb_multi_times.usb_normal_test.usb_ccid.with_usb_hub.one_usb_dev.piix4-uhci.q35: CANCEL: Test cancelled due to --dry-run (0.00 s)

LGTM,
Acked-by xuwei@redhat.com

**Comment by vivianQizhu** [comment]
Thanks @michiboo and @XueqiangWei

**Pull request merged**